multigeiger/multigeiger.ino

```diff
@@ -354,6 +355,27 @@ void setup()
     while (!Serial) {};
   }
 
+  // build revString
+  strcat(revString,software_version);
+  strcat(revString," ");
+  strcat(revString,software_version_date);
+  
+#if SEND2LORA
+  // build LoRa software version
+  char loravers[10];
+  int i = 0;
+  char *p = (char *)software_version;
+  while (*p != '\0') {
+    if (*p != '.') {
+      loravers[i++] = *p;
+    }
+    p++;
+  }
+  loravers[i] = '\0';
+  lora_software_version[0] = atoi(loravers)/100;
+  lora_software_version[1] = atoi(loravers)%100;
```

Comment:
that only works correctly for versions like:
- 1.00
- 1.01
- 1.99

but not for:
- 1.1
- 1.2
- 1.3.1 (in case we would use x.y.z)

Reply:
You are right. To make it easy I wanted to use always 2 digits for the lower number.

Reply:
Then better put a comment into the source if you don't want to break the code.

It's a bit unusual though, normally 1.1 or 1.1.1 should be valid as a version number.

Reply:
Sorry, I guess i clicked the wrong button.

Nice idea, to use xyyz in one 16 bit variable. I agree, we never would use more than 15 big versions and more than 15 patches either. So I will change the versioning system in one of the next PRs.
Thanks for this solution.